src/js/views/patients/sidebar/action/action-sidebar-attachments-views.js

```diff
@@ -0,0 +1,57 @@
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import { alphaSort } from 'js/utils/sorting';
+
+import 'scss/modules/sidebar.scss';
+
+import './action-sidebar.scss';
+
+const AttachmentView = View.extend({
+  className: 'u-margin--t-16',
+  template: hbs`
+    <a class="action-sidebar__attachment-filename" target="_blank" href={{path}}>{{filename}}</a>
+    <div>
+      <a class="action-sidebar__attachment-download" href={{path}} download>
+        {{far "download"}} <span>{{ @intl.patients.sidebar.action.attachmentsViews.attachmentView.downloadText }}</span>
+      </a>
+    </div>
+  `,
+  templateContext() {
+    return {
+      filename: this.model.get('path').split('/').pop(),
+    };
+  },
+});
+
+const AttachmentsCollectionView = CollectionView.extend({
+  childView: AttachmentView,
+  viewComparator(viewA, viewB) {
+    return alphaSort('desc', viewA.model.get('created_at'), viewB.model.get('created_at'));
+  },
+});
+
+const AttachmentsView = View.extend({
+  className: 'u-margin--t-24',
+  template: hbs`
+    <div class="sidebar__attachments">
+      <h3 class="sidebar__heading">
+        {{far "paperclip"}}<span class="u-margin--l-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.attachmentsHeadingText }}</span>
```

Comment:
Missed that one, just updated in the [latest push](https://github.com/RoundingWell/care-ops-frontend/compare/68e0bad7286fe43133101c713b6dfcf1024cb3d8..94513d9123ff0fc3bb004d21722f66285224fbce).

Updated the `src/js/views/patients/sidebar/action/action-sidebar-activity-views.js` as well, since that was also moved from the `actionSidebarTemplate`.